src/liveBlock.ts

```diff
@@ -21,6 +30,26 @@ type LiveBlock = {
 
 // ----- Functions ----- //
 
+const serialiseLiveBlock = ({
+    id,
+    isKeyEvent,
```

Comment:
I think there's another type of block `isSummaryEvent` which we currently style to look like keyEvents in the templates. Maybe we should add this to the model?

Reply:
Wasn't aware of this, do you have an example of what that looks like? 🤔

Reply:
https://www.theguardian.com/world/live/2020/may/20/coronavirus-live-news-brazil-sees-record-daily-deaths-as-world-bank-warns-60m-to-fall-into-extreme-poverty?page=with:block-5ec5aece8f08a1782fa7e955#block-5ec5aece8f08a1782fa7e955

It looks just like a key event. You can see them by going to capi and searching `is-summary`. It looks like those classes are only added in the the big html fields and not in the blocks api. Maybe in the blocks api they are always set to be `key-events`

Reply:
It *looks* like the blocks API provides `attributes.summary: true` for those, so we can probably use that? Seems like the changes we have to make for them is to added the headline `"Summary"` and include them in the key events (which also means adding key events styles)?

Reply:
I think you can have a block that is either “Summary”, “Key Event” or both. Both look the same (yellow title and top border colour) but only key events go into the bullet point links at the top.

Reply:
Yeah spoke too soon, *some* of them have `attributes.summary: true`, others just have `attributes.keyEvent: true`. So what you're saying is that they're *not* a key event, so don't appear in the key events list, but we *do* apply the same styles as key events?

Reply:
Yes that's my understanding. We added that logic to the templates a while ago @benwuersching

Reply:
Ok, good to know. Unrelated to this PR so I've created a ticket.